flow/flow.go

```diff
@@ -1286,20 +1127,22 @@ func (f *Flow) GetFieldString(field string) (string, error) {
 	}
 
 	switch name {
-	case "Link":
-		return f.Link.GetStringField(fields[1])
-	case "Network":
-		return f.Network.GetStringField(fields[1])
+	case "Link", "ETHERNET":
+		if f.Link != nil {
```

Comment:
future enhancement: if you implement `GetFieldString()` so that it handles `nil` pointer you can avoid having to handle `nil` (`if xxx != nil ...`) again and again before calling the method.

Reply:
I would prefer to let the caller handle the `if xxx != nil` case (otherwise we always run the != nil test even if it's not useful)